virttest/ppm_utils.py

```diff
@@ -10,6 +10,10 @@
 import re
 import glob
 import logging
+
+from six.moves import reduce
```

Comment:
use try ... except to import mapping reduce in python3 although it is removed in python3

Reply:
on python 3 ,reduce is moving into functools module
from functools import reduce